service/application/src/main/java/com/solace/maas/ep/event/management/agent/cli/MultipartFileWrapper.java

```diff
@@ -0,0 +1,61 @@
+package com.solace.maas.ep.event.management.agent.cli;
```

Comment:
The existing file import functionality requires an input object of type MultipartFile that is used to get the InputStream for the scan file. This wrapper class was created to do just that and nothing else.